patches/@stardust-collective+dag4-core+1.0.5.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/node_modules/@stardust-collective/dag4-core/cjs/cross-platform/clients/state-storage-db.js b/node_modules/@stardust-collective/dag4-core/cjs/cross-platform/clients/state-storage-db.js
+index d560918..fdb2d0a 100644
+--- a/node_modules/@stardust-collective/dag4-core/cjs/cross-platform/clients/state-storage-db.js
++++ b/node_modules/@stardust-collective/dag4-core/cjs/cross-platform/clients/state-storage-db.js
+@@ -19,11 +19,11 @@ class StateStorageDb {
+         }
+         this.keyPrefix = prefix;
+     }
+-    set(key, value) {
+-        this.storageClient.setItem(this.keyPrefix + key, JSON.stringify(value));
++    async set(key, value) {
```

Comment:
Nice - I think the changes to this file were what was missing from my branch to get this working.